.gitignore

```diff
@@ -45,4 +45,6 @@ Thumbs.db
 src/api.config.ts
 version.js
 yarn-error.log
-.vscode
\ No newline at end of file
+.vscode
+/temp/*
```

Comment:
Were `/temp/` and `*.bat` only for your development? They don't seem to be necessary for the master branch.